### DISCUSSION TITLE
Getting Started with a new Open source project

### DISCUSSION THREAD

**Discussion by Nikhil8557** [community/community#108919] [Other Feature Feedback, Questions, & Ideas]
### Hi there,

> Maybe it's too long to read but please try to do it 😁

 Are you passionate about making a real difference in the world? Do you care about protecting lives and communities from the devastating impact of cyclones? If so, then I have an exciting opportunity for you! I'm working on a powerful project to develop an AI model for predicting cyclones and their impact in India. This model has the potential to save countless lives by enabling:

-  Earlier evacuations: With accurate predictions, people can get to safety before the storm hits, minimizing casualties and losses

-  Better resource allocation: Knowing the predicted path and intensity of a cyclone allows authorities to allocate resources more effectively, ensuring aid reaches those in need quickly. 

- Improved preparedness: Timely warnings give communities time to prepare for the storm, minimizing damage and disruption. But this project can't reach its full potential alone. That's where you come in! 

We're open-sourcing the AI model, building a community of passionate individuals like you who want to contribute their skills and expertise. Here are some ways you can get involved:
 

- [ ] ●Data scientists: Help us improve the model's accuracy by contributing to data collection, cleaning, and analysis. Developers: Collaborate on building and maintaining the open-source codebase. 

- [ ] ●Meteorologists/Disaster management experts: Share your knowledge and insights to refine the model and its interpretation. 

- [ ] ●Anyone with a passion for making a difference: Share your ideas, feedback, and enthusiasm! No matter your background or skill set, there's a way you can contribute. Together, we can build a powerful tool that saves lives and protects communities from the devastating effects of cyclones. Join us today! 

Visit our [website](https://github.com/Nikhil8557/SAGARai) to learn more and get involved. Let's make a difference, together! 

Sincerely, Project lead, SAGARai

 P.S. Share this message with anyone who might be interested in joining our mission! The more, the merrier (and the safer)!
![image-3](https://github.com/community/community/assets/71388692/32baa2a8-b180-4149-bb07-ef6f56b70b7a)

**Comment by Akash1134** [comment]
Hi @Nikhil8557 , we’re glad you’re here 😀

Thanks for sharing your (project/website/etc) with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
